test/local/routes/account_devices.js

```diff
@@ -78,16 +78,16 @@ function runTest (route, request, assertions) {
 
 describe('/account/device', function () {
   var config = {}
-  var uid = uuid.v4('binary')
-  var deviceId = crypto.randomBytes(16)
+  var uid = uuid.v4('binary').toString('hex')
+  var deviceId = crypto.randomBytes(16).toString('hex')
```

Comment:
Seems like `crypto.randomBytes().toString('hex')` is used in a lot in tests. Maybe be useful to pull into test helpers.